tests/realtime/realtimeUtils.js

```diff
@@ -216,5 +216,21 @@ describe('realtime utils Tests:', () => {
         }
       });
     });
+
+    describe('attachAspectSubject tests', () => {
+      it('useSampleStore = true', () => {
+        realtimeUtils.attachAspectSubject(looksLikeSampleObjNA, true)
```

Comment:
we need more tests for this. Can we do a test where the 
1) cache is on, and the subject and aspect are attached?
2) cache is off, and the subject and aspect are attached?